ibm_catalog.json

```diff
@@ -117,9 +136,16 @@
               "key": "prefix",
               "required": true
             },
+            {
+              "key": "region",
+              "type": "string",
+              "description": "Region for OCP Cluster.",
+              "virtual": true,
+              "default_value": "us-south"
```

Comment:
default values should not be in the json! Defaults should be managed in the terraform code only

Reply:
Its a virtual input, hence a default value is required.